integrations/auth0/client.go

```diff
@@ -79,9 +82,42 @@ func connStatus(i *integration) sdkintegrations.OptFn {
 	})
 }
 
-// TODO(INT-113): Implement
-func connTest(_ *integration) sdkintegrations.OptFn {
+// connTest is an optional connection test provided by the integration
+// to AutoKitteh. It is used to verify that the connection is working
+// as expected. The possible results are "OK" and "error
+// https://auth0.com/docs/api/management/v2/stats/get-active-users
```

Comment:
Given that this URL isn't connected to the function description above, maybe move it directly above line 102, or clarify in the function description that it's based on this API call.